readme.md

```diff
@@ -8,7 +8,8 @@ For the following quiz, there will be a series of questions or directions follow
 What is Git, what problem does it solve?
 
 ```
-answer goes here
+
+Git is an online resource that allows people to share and edit eachothers codes. 
 
 ```
 
```

Comment:
Git is version control software, but is easily confused with GitHub. Git solves problems of organizing different file versions, which facilitates collaboration.